hive/datadog_checks/hive/data/conf.yaml.example

```diff
@@ -86,6 +86,27 @@ instances:
     #
     # rmi_registry_ssl: false
 
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
+## service - mandatory - Name of the service that generated the log
+## source  - mandatory - Attribute that defines which Integration sent the logs
+## sourcecategory - optional - Multiple value attribute. Used to refine the source attribute
+## tags: - optional - Add tags to the collected logs
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: /tmp/<USER>/hive.log
```

Comment:
Looks like this is the old path. Should we rather set the default to `/tmp/hive/hive.log`?
https://cwiki.apache.org/confluence/display/Hive/Configuration+Properties#ConfigurationProperties-QueryandDDLExecution

Reply:
Actually it seems that most of the time the user starting hive is `hive`, so the location is `/tmp/hive/hive.log` = `/tmp/<USER>/hive.log`. I think we should have a generic path here

Reply:
Also, in Hive 2.3.2, we still have the log file in `/tmp/<USER>/hive.log`

Reply:
👍